packages/core/nuxt-module/src/helpers/merge.ts

```diff
@@ -0,0 +1,7 @@
+import { mergeWith, isArray } from 'lodash';
```

Comment:
So we've made the ts files but there are no types at all? 🤔

Reply:
Types added